src/actions/matching.js

```diff
@@ -0,0 +1,11 @@
+import * as constants from '../constants/matching';
+import { matching } from '../data/index.json';
```

Comment:
could you use `data.json` instead of `data/index.json`?

Reply:
In the next commit, we will get data from the Firebase cloud. And we will not use this file